### ISSUE TITLE
Webpack 5

### ISSUE THREAD

**Issue by VIKTORVAV99** [electricitymap/electricitymap-contrib#4091]
### Webpack 5 migration tracker

- [x] Update css loaders to highest version that support webpack 4 #4089 
- [x] Update webpack-config.js #4089 
- [x] #4090
- [x] Upgrade to Webpack 5 #4105
- [x] Update loaders and plugins latest versions.

**Comment by madsnedergaard** [comment]
This is super cool - but before spending way too much time on it, should we consider if we want to stick with webpack vs one of the newer alternatives? :)

**Comment by VIKTORVAV99** [comment]
@madsnedergaard I'd say sticking with webpack is the best way forward, it's by far the most supported and documented bundler out there. It's also highly configurable and has tons of useful plugins. 

Webpack 5 is a considerable improvement over 4 when it comes to internals and speed and if we want to output native ESM bundles it has experimental support that too.

Planning ahead I think Webpack 5 and later Webpack 6 is the way forward but I'd be happy to discuss it.

In either case the upgrades in this issue is something we should do in either case as our dependencies are starting to get very outdated.

**Comment by VIKTORVAV99** [comment]
@madsnedergaard turns out this was easier than I expected!

![image](https://user-images.githubusercontent.com/30777521/167031880-54460892-0bec-416d-99a0-cb103612844c.png)

Now this depends on #4089 so can't create a separate PR for it (yet) but let me know if you just want me to update that one!
